pkg/agent/openflow/pipeline.go

```diff
@@ -527,13 +531,24 @@ func (c *client) connectionTrackFlows(category cookie.Category) []binding.Flow {
 				Action().CT(false, connectionTrackTable.GetNext(), CtZone).NAT().CTDone().
 				Cookie(c.cookieAllocator.Request(category).Raw()).
 				Done(),
+			connectionTrackTable.BuildFlow(priorityNormal).MatchProtocol(binding.ProtocolIPv6).
+				Action().CT(false, connectionTrackTable.GetNext(), CtZoneV6).CTDone().
```

Comment:
just to confirm, the NAT action is note added by intention as IPv6 doesn't need NAT?

Reply:
Service cluster IP still NATs the pod IP, even with IPv6. Right?

Reply:
that's my question, I'm not sure the answer. If we need to NAT cluster IP for IPv6 as well, why we don't have "NAT()" action here?

Reply:
Indeed. Thanks for noticing